actions/common/search_methods.go

```diff
@@ -0,0 +1,54 @@
+package common
+
+import (
+	"math"
+
+	"github.com/bitcoin-sv/spv-wallet/engine/datastore"
+)
+
+// Count object to use when returning a count of database query results
+type Count struct {
```

Comment:
maybe `PagedResponse` or `PagedModel` would be better name?

Reply:
Changed to PagedResponse